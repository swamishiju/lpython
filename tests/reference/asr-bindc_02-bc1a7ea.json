{
    "basename": "asr-bindc_02-bc1a7ea",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/bindc_02.py",
    "infile_hash": "ce0bbd7a1a17c689c995fb31bc71275c59459120f0cf7defedd1cf33",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-bindc_02-bc1a7ea.stderr",
    "stderr_hash": "af75969fb5ce6ccb563e874497c4e5290f1a3c4d9fd36217b442fd7a",
    "returncode": 2
}