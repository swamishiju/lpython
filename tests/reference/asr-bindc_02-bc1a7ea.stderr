semantic error: Function 'empty_c_void_p' is not declared and not intrinsic
 --> tests/../integration_tests/bindc_02.py:4:17
  |
4 | queries: CPtr = empty_c_void_p()
  |                 ^^^^^^^^^^^^^^^^ 
